nodejs/week1/avg.js

```diff
@@ -0,0 +1,26 @@
+const arguments = process.argv;
+
+let numbers = []; 
+
+for (let arg of arguments) { 
+    let num = Number(arg); 
+    if (!isNaN(num)) { 
+        numbers.push(num);
+    }
+}
```

Comment:
I see you decided to tolerate, and ignore the parameters that are not valid numbers. Great idea. 🧠 

Another option would have been to exit the script with an error in these cases.